user_guide_src/source/libraries/uploaded_files.rst

```diff
@@ -83,9 +83,9 @@ this code and save it to your **app/Controllers** directory:
 
 .. literalinclude:: uploaded_files/002.php
 
-Since the value of a file upload HTML field doesn't exist, and is stored in the
+Since HTML file element has no value attribute, and the upload file is stored in the
 ``$_FILES`` global, only :ref:`rules-for-file-uploads` can be used to validate
```

Comment:
"the upload file is stored in the ``$_FILES`` global"
This is not true or misleading. 
The global [$_FILES](https://www.php.net/manual/en/reserved.variables.files.php) contains all the uploaded file information.